### ISSUE TITLE
About finetune of mandarin model

### ISSUE THREAD

**Issue by Zolewit** [alphacep/vosk-api#437]
I am very satisfied with the results of the model (vosk-model-cn-0.1). It's fast, accurate and small.

I read the documentation of adaptation and issues #299 , but they confuse me.
Can I improve vosk-model-cn-0.1's accuracy (finetune) without your help?
Because I am a newbie to asr, just want to learn, not to disturb you.

**Comment by nshmyrev** [comment]
Hi

Unfortunately no straighforward way for fine-tuning right now, there are example scripts but they are not easy to use for the newbie. I'll keep this open and update you once we have something more detailed.

**Comment by Zolewit** [comment]
Thank you, I will continue to pay attention

**Comment by nshmyrev** [comment]
Actually we have the issue about it #185 with some example links. Lets track the problem there.
